docs/gaia/gaiacli.md

```diff
@@ -277,21 +277,21 @@ On the upcoming mainnet, you can delegate `atom` to a validator. These [delegato
 You can query the list of all validators of a specific chain:
 
 ```bash
-gaiacli query stake validators
```

Comment:
shouldn't these commands remain the same ?

Reply:
no, this is a module rename they should all be upgraded to `staking`